### DISCUSSION TITLE
Use custom "values.yml" values that come inside the Chart

### DISCUSSION THREAD

**Discussion by fabioaraujopt** [argoproj/argo-cd#9677] [Q&A]
We have a set of charts that are shipped to registry not only with the "values.yml" but also with several other "values_override.yml" customizations (eg. values_production.yml)
Is there a way to use argoCD application.yml to use the helm chart values which live inside the chart itself? So the application.yml would install the latest helmChart and would apply the "values.yml + values_production.yml". 

```
  source:
    repoURL: XXXX
    targetRevision: main
    
    # helm specific config
    chart: calendar  # Set this when pulling directly from a Helm repo. DO NOT set for git-hosted Helm charts.
    helm:
      valueFiles:
      - values_production.yml
```

This only works if the values_production.yml are in the same repo as the "application.yml".

**Comment by crenshaw-dev** [comment]
You can either hard-code the values into [the `values` block](https://github.com/argoproj/argo-cd/blob/91de2f375008610be4f9164aaf8991a05b5e23e5/docs/operator-manual/application.yaml#L49) (or template them in using something like Helm) or wait for this feature in 2.5. https://github.com/argoproj/argo-cd/issues/2789

**Reply by fabioaraujopt** [reply]
Is this really being handled by the team?

**Reply by crenshaw-dev** [reply]
It is. Particularly Ishita from Red Hat is working on the draft PR, then other folks will be reviewing it as soon as it's ready.
